docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -2,19 +2,31 @@
 
 #### The Orderbook
 
-The orderbook is the main entry point of the protocol. Clients can use the Orderbook API to create trades, get quotes for trades, get currently solved auction, etc. When you go to [CoW Swap](https://swap.cow.fi) and create a trade, the web site uses the OrderBook API to add the trade to the database. All the data orderbook use is stored in the database.
+The orderbook is the main entry point of the protocol. 
+Clients can use the Orderbook API to create trades, get quotes for trades, get currently solved auction, etc. 
+When you go to [CoW Swap](https://swap.cow.fi) and create a trade, the web site uses the OrderBook API to add the trade to the database. 
+All the data orderbook use is stored in the database.
 
 #### The Autopilot
 
-The autopilot is a service that drives the core protocol functionalities. It is responsible for creating and arbitrating auctions. It communicates with the driver and decides when the auction is solved and when the solution is revealed and finally settled. Besides that, autopilot is responsible for monitoring the solvers and making sure they behave correctly (and slashing them if they don't).
+The autopilot is a service that drives the core protocol functionalities.
+It is responsible for creating and arbitrating auctions.
+It communicates with the driver and decides when the auction is solved and when the solution is revealed and finally settled.
+Besides that, autopilot is responsible for monitoring the solvers and making sure they behave correctly (and slashing them if they don't).
```

Comment:
The `autopilot` is not so much involved in slashing. And while it's true that it stores data for each auction for the winning solver calling it monitoring in this context seems a bit misleading.

Reply:
This is fixed in the other PR (here I just created new lines)